packages/serverpod/lib/src/server/endpoint_dispatch.dart

```diff
@@ -139,6 +139,8 @@ abstract class EndpointDispatch {
       throw EndpointNotFoundException('Endpoint $endpointPath not found');
     }
 
+    if (connector.endpoint.logSessions) session.startLogging();
```

Comment:
Question: Why is this added here? It should be unnecessary as the session will track if logging is enabled/needed (internally).

Reply:
Reverted here: [cec5fad](https://github.com/serverpod/serverpod/pull/2676/commits/cec5fad3d1dececc42a205ca6662da6c4342f464)